projects/storefrontapp-e2e-cypress/cypress/helpers/product-configuration-overview.ts

```diff
@@ -1,17 +1,92 @@
 const continueToCartButtonSelector = 'cx-config-add-to-cart-button button';
+const resolveIssuesLinkSelector =
+  'cx-configure-cart-entry button.cx-action-link';
 
+/**
+ * Navigates to the configured product overview page.
+ *
+ * @param configuratorType - Configuration type
+ * @param productId - Product ID
+ */
+export function goToConfigOverviewPage(configuratorType, productId) {
```

Comment:
I suggest to specify void return type here

Reply:
Done